toolchain/check/testdata/function/declaration/no_prelude/name_poisoning.carbon

```diff
@@ -12,1284 +12,544 @@
 
 library "[[@TEST_NAME]]";
 
-class C {};
+fn F1();
```

Comment:
Make these have different return types so you can detect which is used.
```suggestion
fn F1() -> {};
```

Reply:
Made them take different params.